source/mongodb/crud-and-aggregation-apis.txt

```diff
@@ -664,7 +664,7 @@ Database Command Availability
 
    * - :manual:`aggregate </reference/command/aggregate>`
      - - ``explain``
-       - ``allowDiskUse``
+       - ``allowDiskUse`` *
```

Comment:
This is by no means required, but have you considered using a footnote instead? I think it'll provide a link, not 100% sure though since I've never used one.

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#footnotes

Reply:
I did not know this was a thing. Giving it a try!

Reply:
Well whaddya know, that worked and I like it! Thanks again for the suggestion.